packages/blocks/src/sites.ts

```diff
@@ -85,39 +90,35 @@ export interface SiteAnalytics {
 export interface SiteDesign {
   hide_authors?: boolean;
 }
+
 export const SITE_CONFIG_KEYS = {
   optional: [
-    'domains',
-    'title',
-    'description',
-    'venue',
     'projects',
     'nav',
     'actions',
+    'domains',
     'twitter',
     'logo',
     'logo_text',
+    'logoText',
```

Comment:
```suggestion
    'logoText', // Deprecated, but included in validation
```


I am actually not sure if this should be in here? Shouldn't we warn on it?

Reply:
We filter against these keys before we do any other validation: https://github.com/curvenote/curvenote/pull/345/files/62970e02c90ffce8046149cea36af5081ee7e112#diff-01b0951fb3833dadc72bbf40ff110278796d45b9282ce0487f68b68775785ff2R372

I suppose it would make sense to do this immediately on `readConfig`, the same place where we un-nest frontmatter and warn...